runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsConfig.java

```diff
@@ -88,6 +90,17 @@ default boolean scoutOverlayInRaid()
 
 	@ConfigItem(
 		position = 5,
+		keyName = "ccDisplayStyle",
+		name = "CC and World in scout overlay",
+		description = "Display style of current CC and World in scouting overlay"
```

Comment:
There's not really a reason to abbreviate "clan chat" in the description. Let's write that out to be explicit and clear.